index.html

```diff
@@ -3235,12 +3235,12 @@ <h3>
 
     <ol class="punctuation" type="1">
       <li id="id195">
-        <p its-locale-filter-list="en" lang="en" class="translateme">Translation needed.</p>
+        <p its-locale-filter-list="en" lang="en">All pause or stop punctuation marks can appear at the end of a line, but are not allowed at the start of a line.</p>
         <p its-locale-filter-list="zh-hans" lang="zh-hans">所有点号皆不可出现于行首，但可出现于行尾。</p>
         <p its-locale-filter-list="zh-hant" lang="zh-hant">所有點號皆不可出現於行首，但可出現於行尾。</p>
       </li>
       <li id="id196">
-        <p its-locale-filter-list="en" lang="en" class="translateme">Translation needed.</p>
+        <p its-locale-filter-list="en" lang="en">Although pause or stop punctuation marks may have some layout difference between vertical writing mode and horizontal writing mode, they both don't need to rotate. Read main body to understand what's difference between them.</p>
```

Comment:
May I suggest a few tweaks?
"Although pause or stop punctuation marks may have some layout _differences_ between vertical writing mode and horizontal writing mode, _neither need to be rotated_. Refer to the main text for details on the differences between them."

Reply:
Your modify is very great, thank you very much.